contracts/ERC721A.sol

```diff
@@ -62,19 +75,42 @@ contract ERC721A is Context, ERC165, IERC721, IERC721Metadata, IERC721Enumerable
         _symbol = symbol_;
     }
 
+    /**
+     * @dev Skips the zero index.
+     * This method must be called before any mints (e.g. in the consturctor).
+     */
+    function _initOneIndexed() internal {
```

Comment:
i know there's a desire to be able to set where the collection starts, it seems weird to accomplish that by "burning" a token

Reply:
Other alternatives for consideration:
- Use another uint256 to stores the initial offset. To save gas, it is made immutable. This requires a third argument for the constructor, which may cause confusion, as old versions of ERC721A use the third argument for the batchSize.
- Use another uint128 to stores the initial offset. To save gas, we reduce the resolution for totalBurned to uint128 so that we can pack this new variable with it. May need more changes throughout the code.

Reply:
there's certainly a lot of approaches, just requires a lot of checks/changes throughout the file

Reply:
Removed the 1-index functionality.